src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -945,7 +945,15 @@ public void Close(int timeout)
             if (NetEventSource.IsEnabled) NetEventSource.Exit(this, timeout);
         }
 
-        // Places a socket in a listening state.
+        /// <summary>
+        /// Places a <see cref="Socket"/> in a listening state.
+        /// </summary>
+        public void Listen() => Listen(int.MaxValue);
+
+        /// <summary>
+        /// Places a <see cref="Socket"/> in a listening state.
+        /// </summary>
+        /// <param name="backlog">The maximum length of the pending connections queue.</param>
```

Comment:
> The operating system may choose a reduced limit.

It may also choose a higher one.  It's really just a hint.

Also, any changes to the existing XML comments won't propagate automatically to the docs repo.  It's fine to improve the XML comments, but if there are meaningful changes to the existing ones, we should also submit a PR to the docs repo to improve them accordingly.

Reply:
> pending accept queue

The man page for listen refers to it as pending connections; I think it's reasonable to keep it the same.